app/assets/javascripts/models/data/colors.js.coffee

```diff
@@ -0,0 +1,19 @@
+#= require models/data/xhr_model
+#= require models/data/variable
+
+ns = @edsc.models.data
+
+ns.Colors = do () ->
+
+  class ColorsModel
+    constructor: () ->
+      # Defines the color values for collections
+      @collections = ko.observableArray([
```

Comment:
Instead of this being an array, why don't we change it to just being properties on this class.
```
class ColorsModel
  constructor: ->
    @blue = '#3498DB'
    ...
    ...
```
then in `project.js.coffee` you could do 
```
colorPool = new ValuePool([
     colors.blue,
     colors.orange,
     colors.green,
     colors.red,
     colors.purple
   ])	   
```